src/ad-bidders/prebid/adapters/one-video.ts

```diff
@@ -0,0 +1,38 @@
+import { PrebidAdapter } from '../prebid-adapter';
+
+export class OneVideo extends PrebidAdapter {
+	static bidderName = 'oneVideo';
+
+	get bidderName(): string {
+		return OneVideo.bidderName;
+	}
+
+	prepareConfigForAdUnit(code, { site, pubId }): PrebidAdUnit {
+		return {
+			code,
+			mediaTypes: {
+				video: {
+					playerSize: [640, 480],
+					context: 'instream',
+				},
+			},
+			bids: [
+				{
+					bidder: this.bidderName,
+					params: {
+						site,
+						pubId,
+						video: {
+							playerWidth: 640,
+							playerHeight: 480,
+							mimes: ['video/mp4', 'application/javascript'],
```

Comment:
```suggestion
							mimes: ['video/mp4', 'video/x-flv', 'video/webm', 'video/ogg'],
```
We want only videos here, `application/javascript` would allow VPAIDs which could have specific behaviour or change things and we don't want it to happen.